.nyc_output/processinfo/index.json

```diff
@@ -1 +1 @@
-{"processes":{"a9861d87-c4ff-442d-a585-40eea33ccd48":{"parent":null,"children":[]}},"files":{"C:\\Users\\irfiacre\\Desktop\\Papel\\v0.0\\Papel\\server\\tests\\appTest.js":["a9861d87-c4ff-442d-a585-40eea33ccd48"],"C:\\Users\\irfiacre\\Desktop\\Papel\\v0.0\\Papel\\server\\app.js":["a9861d87-c4ff-442d-a585-40eea33ccd48"],"C:\\Users\\irfiacre\\Desktop\\Papel\\v0.0\\Papel\\server\\routes\\clientRoute.js":["a9861d87-c4ff-442d-a585-40eea33ccd48"],"C:\\Users\\irfiacre\\Desktop\\Papel\\v0.0\\Papel\\server\\middleware\\validator.js":["a9861d87-c4ff-442d-a585-40eea33ccd48"],"C:\\Users\\irfiacre\\Desktop\\Papel\\v0.0\\Papel\\server\\controllers\\user.js":["a9861d87-c4ff-442d-a585-40eea33ccd48"],"C:\\Users\\irfiacre\\Desktop\\Papel\\v0.0\\Papel\\server\\config\\db-config.js":["a9861d87-c4ff-442d-a585-40eea33ccd48"],"C:\\Users\\irfiacre\\Desktop\\Papel\\v0.0\\Papel\\server\\models\\testData.js":["a9861d87-c4ff-442d-a585-40eea33ccd48"],"C:\\Users\\irfiacre\\Desktop\\Papel\\v0.0\\Papel\\server\\tests\\userTest.js":["a9861d87-c4ff-442d-a585-40eea33ccd48"]},"externalIds":{}}
```

Comment:
This folder is still visible on your repository.  Hint: you could create a copy of it on your local machine and delete it before you push to the remote repository. Then you can add it back to your project folder. It won't be tracked this time because you have already added it to your .gitignore.